src/opensynth/models/faraday/gaussian_mixture/prepare_gmm_input.py

```diff
@@ -55,12 +57,21 @@ def prepare_data_for_model(
     Args:
         data (torch.Tensor): data for GMM training.
         vae_module (FaradayVAE): VAE module used for encoding.
+        train_sample_weights: flag whether to train with sample weights.
+        Defaults to false
```

Comment:
Does this default to `False`? looks to me like it's a required arg.

Reply:
Ah yeah I changed my mind about letting it be optional - will update doc strings!